### PR TITLE
[MLU] fix mlu backend unittests for custom device

### PR DESCRIPTION
**Pull request by wufann**
[MLU] fix mlu backend unittests for custom device

### CODE REVIEW

**Review by ShawnNew — APPROVED** [review]
👍

File: backends/mlu/tests/unittests/test_where_op_mlu.py [hunk 1]
```diff
@@ -298,24 +291,24 @@ def test_dygraph_api_broadcast_8(self):
         self.__test_where_with_broadcast_dygraph(cond_shape, a_shape, b_shape)
 
     def test_where_condition(self):
-        data = np.array([[True, False], [False, True]])
+        data = np.array([[True, False], [False, True]]).astype(np.float32)
```
Comment (ShawnNew) [inline backends/mlu/tests/unittests/test_where_op_mlu.py]: 直接删除吧

**Review by ShawnNew — CHANGES_REQUESTED** [review]

File: backends/mlu/tests/unittests/test_where_op_mlu.py (hunk 1 above)
Comment (wufann) [reply backends/mlu/tests/unittests/test_where_op_mlu.py]: done

**Review by ShawnNew — APPROVED** [review]

**Review by ShawnNew — APPROVED** [review]
👍

**Pull request merged**